### DISCUSSION TITLE
Request for Gamepad Option Steam Deck/Steam Controller

### DISCUSSION THREAD

**Discussion by mxp83** [LizardByte/community#290] [Sunshine]
Requesting that Steam Deck be an option for Sunshine Gamepad.  I have REWASD on my Host PC and they recently released an update which allows the Steam Deck be recognized as a controller for configuration, but only if REWASD is installed on Windows in the Steam Deck.

**Comment by ReenigneArcher** [comment]
What you are requesting is not very clear. Emulating a steam deck gamepad on a Windows host running Sunshine is not possible currently as we rely on Vigembus, which only allows Xbox 360 and DS4 gamepad emulation. I'm closing this since I don't think it's feasible at the moment.
